__mocks__/cairo/helloSierra/hello.cairo

```diff
@@ -328,6 +329,22 @@ mod HelloStarknet {
         testbet::read()
     }
 
+    //#[external]
```

Comment:
leftover or?

Reply:
Yea, that method was required by ex compiler to include the struct in ABI now it is fixed.